binance/threaded_stream.py

```diff
@@ -17,7 +17,7 @@ def __init__(
 
         """
         super().__init__()
-        self._loop: asyncio.AbstractEventLoop = asyncio.get_event_loop() if asyncio.get_event_loop().is_running() else asyncio.new_event_loop()
+        self._loop: asyncio.AbstractEventLoop = asyncio.get_event_loop()
```

Comment:
@pcriadoperez shouldn't we use here the helper `get_loop`?  it has some extra protections in case get_event_loop throws an exception